Project.toml

```diff
@@ -5,9 +5,12 @@ version = "0.2.0"
 
 [deps]
 Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
+JSOSolvers = "10dff2fc-5484-5881-a0e0-c90441020f8a"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+NLPModelsTest = "7998695d-6960-4d3a-85c4-e1bceb8cd856"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Weave = "44d3d7a6-8a23-5bf8-98c5-b353f8df5ec9"
 YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
```

Comment:
It doesn't make sense. It's the Project.toml in the same folder of index.jmd that should be updated